pyxem/signals/diffraction2d.py

```diff
@@ -1883,9 +1880,6 @@ def get_azimuthal_integral1d(
         azimuth_range : None or (float, float)
             The azimuthal range over which to perform the integration. Default is
             from -pi to pi
-        wavelength : None or float
```

Comment:
I have been a bit out-of-the-loop with regards to pyFAI: I'm guessing the wavelength "move" to `set_ai` was done in a previous pull request? Or it was previously possible to do it in two separate ways?

Reply:
It was done previously and then just never changed.  The deprecation is still probably a good idea